src/Web3/Version.elm

```diff
@@ -0,0 +1,96 @@
+module Web3.Version exposing (api, getNode, getNetwork, getEthereum)
+
+{-| Version allows one to check the various library, protocol, & network versions one is interacting with. [Web3
+documentation on Version](https://github.com/ethereum/wiki/wiki/JavaScript-API#web3versionapi).
+
+
+# Versions
+
+@docs api, getNode, getNetwork, getEthereum
+
+-}
+
+import Web3 exposing (Error)
+import Web3.Types exposing (Hex, CallType(..))
+import Web3.Decoders exposing (expectString)
+import Json.Encode as Encode
+import Task exposing (Task)
+
+
+-- VERSIONS
+
+
+{-| The ethereum js api version
```

Comment:
Using Traditional Elm documentation. Perhaps we can generate Elm style docs with Evan's helper, rebrand, and make a wiki/github site for our docs.